hawkbit-extension-artifact-repository-gcs/pom.xml

```diff
@@ -0,0 +1,60 @@
+<!--
+
+    Copyright (c) 2015 Bosch Software Innovations GmbH and others.
```

Comment:
Correct me when I'm wrong but I am not sure if you're a fellow of `Bosch Software Innovations GmbH`. 
If not please change the title to give proper credit to either `Copyright (c) 2019 Rico Pahlisch and others` or `Copyright (c) 2019 <YOUR COMPANY> and others`. Depending on whether or not your employment contract stipulates that intellectual property developed by you is the property of your company. 

Don't forget to add the new header template [here](https://github.com/eclipse/hawkbit-extensions/tree/master/licenses).